pkg/util/kernel/kernel.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kernel
+
+import (
+	"fmt"
+	"io/ioutil"
+	"strings"
+
+	utilexec "k8s.io/utils/exec"
+)
+
+// Handler can handle the current installed kernel.
+type Handler interface {
+	GetKernelVersion() (string, error)
+}
+
+// LinuxKernelHandler implements kernel.Handler interface.
+type LinuxKernelHandler struct {
```

Comment:
This can be unexported `linuxKernelHandler`

Reply:
type LinuxKernelHandler is referenced by ipvs.LinuxKernelIPVSHandler